lib/SILOptimizer/Transforms/AllocBoxToStack.cpp

```diff
@@ -693,9 +695,19 @@ static bool rewriteAllocBoxAsAllocStack(AllocBoxInst *ABI) {
                          ABI->getBoxType(), ABI->getModule().Types, 0));
   auto Loc = CleanupLocation(ABI->getLoc());
 
+  auto *deb = deba.get(ABI->getFunction());
```

Comment:
No need to fix this in the old pass. As long as we can successfully build the stdlib with this and the bootstrapping CI job does not fail, we are fine.